bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1159 @@
+## @section Global parameters
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+
+## @param global.imageRegistry Global Docker image registry
+## @param global.imagePullSecrets Global Docker registry secret names as an array
+## @param global.storageClass Global StorageClass for Persistent Volume(s)
+##
+global:
+  imageRegistry:
+  ## E.g.
+  ## imagePullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  imagePullSecrets: []
+  storageClass:
+## @section Common parameters
+## @param kubeVersion Override Kubernetes version
+##
+kubeVersion:
+## @param nameOverride String to partially override common.names.fullname
+##
+nameOverride:
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride:
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+
+## @section Traffic Exposure Parameters
+
+## @section consourse Parameters
+
+concourse:
+  ## Bitnami Concour image
+  ## ref: https://hub.docker.com/r/bitnami/redis/tags/
+  ## @param concourse.image.registry image registry
+  ## @param concourse.image.repository image repository
+  ## @param concourse.image.tag image tag (immutable tags are recommended)
+  ## @param concourse.image.pullPolicy image pull policy
+  ## @param concourse.image.pullSecrets image pull secrets
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/concourse
+    tag: 7.3.2-debian-10-r7
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+
+web:
+  ## @param web.enabled Enable web
+  enabled: true
+  ## @param web.replicaCount Number of web replicas to deploy
+  ##
+  replicaCount: 1
+
+  ## @param web.args Args of the startup command for the web component.
+  ##
+  args: ["web"]
```

Comment:
For consistency with other charts, we leave `args` and `command` for fully custom commands. The default command in the chart should go to the template. Check other charts for more examples

Reply:
Not done